sbi/src/error.rs

```diff
@@ -24,6 +24,8 @@ pub enum Error {
     AlreadyStarted = -7,
     /// Some of the given counters have already been stopped.
     AlreadyStopped = -8,
+    /// The buffer passed as a parameter is invalid.
+    InvalidBuffer = -9,
```

Comment:
From the name (and associated comment), I'd assume this was an alias for `InvalidAddress`, but it looks like you're using it for "insufficient buffer length". Maybe change the name to reflect that? Or just use `InvalidParam`?

Reply:
I changed the name, I'd prefer to not piggy back all errors on InvalidParam and give more details through a more specific error.